scripts/ibazel.js

```diff
@@ -0,0 +1,10 @@
+// Temporary wrapper script which makes sure that the proper, locally-installed, binary is used when
+// running ibazel. Currently ibazel has a bug where it only uses the globally-installde version.
+// See: https://github.com/bazelbuild/bazel-watcher/issues/339
+const shelljs = require('shelljs');
+const path = require('path');
+const localIbazelBinary = path.resolve('./node_modules/.bin/ibazel');
```

Comment:
Will that work on Windows where Bash files are not usually set up in the `PATHEXT`? I think we might be able to just do:

```
// Not using the Bash syntax though. Just an example of what could work:
$(yarn bin)/ibazel -bazel_path=... args
```

Reply:
It worked for me on Windows through cmd.exe.

Reply:
Okay. Sounds good. We'd see if it breaks on your or my machine.